pkg/netlink/netlink.go

```diff
@@ -154,6 +155,20 @@ func (n *netlinkType) EnableLooseModeReversePathFilter(interfaceName string) err
 	return errors.Wrapf(err, "unable to update rp_filter proc entry for interface %q", interfaceName)
 }
 
+func (n *netlinkType) GetReversePathFilter(interfaceName string) ([]byte, error) {
+	path := "/proc/sys/net/ipv4/conf/" + interfaceName + "/rp_filter"
+
+	existing, err := os.ReadFile(path)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to read proc entry for interface %q", interfaceName)
+	}
+
+	// Ignore leading and terminating newlines
+	existing = bytes.Trim(existing, "\n")
+
+	return existing, nil
```

Comment:
How about return `string` for convenience?

```suggestion
	return string(bytes.Trim(existing, "\n")), nil
```

Reply:
It is aligned with the setSysctl APIs. 

https://github.com/submariner-io/submariner/blob/ba52b8b2bb3494cb4e9c781d12f1ff68bd7b2097/pkg/netlink/netlink.go#L174